src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -332,6 +349,9 @@ extern "C" __declspec(dllexport) HRESULT __stdcall CallApplicationMain(PCALL_APP
     size_t cchTrustedPlatformAssemblies = 0;
     LPWSTR pwszTrustedPlatformAssemblies = nullptr;
 
+    // set handler for security-enhanced CRT functions
+    _set_invalid_parameter_handler(invalid_parameter_handler);
```

Comment:
Is it possible to find out the scope of ```_set_invalid_parameter_handler``` call and if it affects the thread or the whole process (as opposed to just current execution frame), restore the previous handler? We don't want to always print and swallow errors.